src/styles/base.scss

```
@@ -103,10 +103,18 @@ a:hover{
   }
 }
 
+img.fail{
```

```suggestion
img.fail {
```
Will be awesome if you will choose rather you have this space before the brace in styles or not. Because most of this document have it. You have different style of spacing in your own PR.

---

Yes, accidentally. It should have a space, I also recommend adding a linter for scss
I avoid formatting the old files, so i do it manually, so sometimes miss it.